package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "js-restaurant",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
**REQUIRED**
Necessary run common commands like build, development should be in package.json inside `scripts'. Here is a basic example ([screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png))